website/docs/secrets/setup-sops.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Setup Sops
+hide_title: true
+---
+
+
+# Managing secrets with Sops
+
+Mozilla SOPS is an open source tool for managing secrets. It uses a client-server approach to encrypting and decrypting the data key.
```

Comment:
If any of this text is copied from the official docs, it's good to link the `Mozilla SOPS` text to it.